velox/common/encode/Base64.cpp

```diff
@@ -81,13 +87,19 @@ constexpr const Base64::ReverseIndex kBase64UrlReverseIndexTable = {
     255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
     255};
 
-constexpr bool checkForwardIndex(
+// Validate the character in charset with ReverseIndex table
+static constexpr bool checkForwardIndex(
```

Comment:
remove `constexpr`

Reply:
The convention is to use anonymous namespace instead of static for such functions.
Anonymous namespace should be at the very top.
See for example: https://github.com/facebookincubator/velox/blob/0fe4fb951ada793c04a10b806a91dfbb52828314/velox/connectors/hive/SplitReader.cpp#L30-L64

Reply:
it is directly used by static assertion, so couldn't be removed